scripts/scenarios/cloud6/qa/ssl-insecure/qa-scenario-1a-ha-kvm-swift.yaml

```diff
@@ -190,10 +190,12 @@ proposals:
       ceilometer-server:
       - cluster:services
       ceilometer-swift-proxy-middleware:
-      - cluster:services
+      - "@@controller1@@"
```

Comment:
this was already addressed in https://github.com/SUSE-Cloud/automation/pull/840.